libmount/src/libmount.h.in

```diff
@@ -31,6 +31,7 @@ extern "C" {
 #include <stdio.h>
 #include <mntent.h>
 #include <sys/types.h>
+#include <stdint.h>
```

Comment:
Yes, I thought about it too. The `stdint.h` is a "must-have" (due to u64); boolean is just optimization, but maybe it's time to use it in the API too.

In structures, I prefer bit-arrays (unsigned int foo: 1), but it would probably be better to update to boolean for the rest. We can prepare a separate PR for this purpose; there are more places to use boolean and not related to statmount/listmount.

Reply:
I don't think we can change it in existing APIs for ABI reasons, but for the new `statmnt_disable_fetching()` it could be used.